### PR TITLE
Jira AT-1046 and Jira AT-1151

### PR DESCRIPTION
**Pull request by PieterJP**
Jira AT-1046 and AT-1151:
SpectrometerHDF5OutputFile.h and cpp:
1. Add noise diode temperature vs frequency tables from files for each band and for 5 and 20 kelvin
2. Add write csv to HDF

Add noise diode noise temperature vs frequency tables for band 1 and band2, 5 K and 20 K, LCP and RCP from files in NoiseDiode directory.

SpectrometerHDF5OutputFile.h and cpp:
1. Remove observation information from a file.
2. Add attributes for:
3. antenna name
4. antenna diameter
5. half power beamwidth
6. antenna longitude
7. antenna latitude
8. antenna altitude
9. observer
10. Conversion from DMS to degrees

### CODE REVIEW

File: SpectrometerHDF5OutputFile.cpp [hunk 1]
```diff
@@ -3468,3 +3654,94 @@ void cSpectrometerHDF5OutputFile::addAttenuationADCChan1(int64_t i64Timestamp_us
...
+        lineCount++;
+    }
+    string row;
+    const string seperator = ",";
+    //string item;
+    const int dimension = 2;
+    double data[lineCount][dimension];
+
+    int entries = 0;
+    //while (getline(inFile, row))
```
Comment (rohanschwartz) [inline SpectrometerHDF5OutputFile.cpp]: You should remove this

File: SpectrometerHDF5OutputFile.cpp [hunk 2]
```diff
@@ -3069,46 +3172,129 @@ void cSpectrometerHDF5OutputFile::addObservationStatus(int64_t i64Timestamp_us,
...
 
-void cSpectrometerHDF5OutputFile::setAntennaLongitude(const string &strAntennaLongitude_deg)
+void cSpectrometerHDF5OutputFile::setObservationInfo(const string &strObservationInformation)
 {
     boost::shared_lock<boost::shared_mutex> oLock(m_oAppendDataMutex);
-    //Only a single set of values
-    sprintf(m_oAntennaConfiguration.m_chaAntennaLongitude_deg, "%s", strAntennaLongitude_deg.c_str());
-}
+    // Antenna\_info:\_Latitude:\_05:45:02.4696\_Longitude:\_00:18:17.9172\_Altitude:\_116\_Diameter:\_32\_Name\_of\_observer:\_ATP_Operator
+    string strAntennaName           = "TDB";  // Must still be addes in SCS
```
Comment (rohanschwartz) [inline SpectrometerHDF5OutputFile.cpp]: Maybe these should just be empty rather than `TBD`

**Review by rohanschwartz — APPROVED** [review]
LGTM, just the two minor comments to consider.

File: SpectrometerHDF5OutputFile.cpp (hunk 1 above)
Comment (PieterJP) [reply SpectrometerHDF5OutputFile.cpp]: Removed.

**Review by rishadebrahim — APPROVED** [review]
LGTM

**Pull request merged**